src/TikzPictures.jl

```diff
@@ -182,23 +182,45 @@ function latexerrormsg(s)
 end
 
 function save(f::PDF, tp::TikzPicture)
-    # Generate the .tex file and make pass along any possible errors
-    foldername = dirname(f.filename)
-    if isempty(foldername)
-        foldername = "."
+    # PDF's are intermediate files for SVG's, so if this function 
+    # is being called inside the save(f::SVG, tp::TikzPicture) function,
+    # the f.filename string would have a temp directory path attached to it
+    # ex - tmp3208yfe/test
+    ind = findfirst(isequal('/'), f.filename)
```

Comment:
why is this bit needed now?

Reply:
Was doing something earlier with it. It isn't needed. I'll remove it with this next commit.